src/test/ui/lint/rfc-2383-lint-reason/no_ice_for_partial_compiler_runs.stdout

```diff
@@ -1,14 +1,12 @@
 #![feature(prelude_import)]
 #![no_std]
-// This ensures that ICEs like rust#94953 don't happen
-// check-pass
-// compile-flags: -Z unpretty=expanded
-
-#![feature(lint_reasons)]
 #[prelude_import]
 use ::std::prelude::rust_2015::*;
 #[macro_use]
 extern crate std;
+// This ensures that ICEs like rust#94953 don't happen
```

Comment:
Why was this moved?

Reply:
IDK, I just ran rustc and blessed the tests. I'm guessing that something related to this was merged meanwhile? The previous position put in the prelude, which seems wrong now that I see it :sweat_smile: If the tests pass I think it's fine.